server/scripts/data.js

```diff
@@ -0,0 +1,35 @@
+// speech bubbles and among us characters
+const canvasElementsData = [
+  {
+    imageUrl: 'https://imgur.com/pphL0E3'
```

Comment:
I'm storing all of these images in imgur for now, we should replace these URLs with S3 when we're ready.